.gitignore

```diff
@@ -13,6 +13,7 @@ target/
 
 # Some of our bench outputs
 /tfhe/benchmarks_parameters
+/tfhe-zk-pok/benchmarks_parameters
```

Comment:
what's that ?

Reply:
running the benchmarks will create this folder which should not be versioned.